website/source/docs/builders/osc-bsu.html.md

```diff
@@ -0,0 +1,355 @@
+    that is *launched* to create the OMI. These tags are *not* applied to the
+    resulting OMIS unless they're duplicated in `tags`. This is a [template
+    engine](../templates/engine.html), see [Build template
+    data](#build-template-data) for more information.
+
+- `run_volume_tags` (object of key/value strings) - Tags to apply to the
+    volumes that are *launched* to create the OMI. These tags are *not* applied
+    to the resulting OMIS unless they're duplicated in `tags`. This is a
+    [template engine](../templates/engine.html), see [Build template
+    data](#build-template-data) for more information.
+
+- `security_group_id` (string) - The ID (*not* the name) of the security
+    group to assign to the VM. By default this is not set and Packer will
+    automatically create a new temporary security group to allow SSH access.
+    Note that if this is specified, you must be sure the security group allows
+    access to the `ssh_port` given below.
+
+- `security_group_ids` (array of strings) - A list of security groups as
+    described above. Note that if this is specified, you must omit the
+    `security_group_id`.
+
+- `shutdown_behavior` (string) - Automatically terminate VMs on
+    shutdown in case Packer exits ungracefully. Possible values are "stop" and
+    "terminate", default is `stop`.
+
+- `skip_region_validation` (boolean) - Set to true if you want to skip
+    validation of the region configuration option. Default `false`.
+
+- `snapshot_groups` (array of strings) - A list of groups that have access to
+    create volumes from the snapshot(s). By default no groups have permission
+    to create volumes from the snapshot(s). `all` will make the snapshot
+    publicly accessible.
+
+- `snapshot_users` (array of strings) - A list of account IDs that have
+    access to create volumes from the snapshot(s). By default no additional
+    users other than the user creating the OMIS has permissions to create
+    volumes from the backing snapshot(s).
+
+- `snapshot_tags` (object of key/value strings) - Tags to apply to snapshot.
+    They will override OMIS tags if already applied to snapshot. This is a
+    [template engine](../templates/engine.html), see [Build template
+    data](#build-template-data) for more information.
+
+- `source_omi_filter` (object) - Filters used to populate the `source_omi` field.
+  - `filters` (map of strings) - filters used to select a `source_omi`.
+  - `owners` (array of strings) - Filters the images by their owner. You may specify one or more Outscale account IDs, "self" (which will use the   account whose credentials you are using to run Packer). This option is required for security reasons.
+  
+    Example:
+
+    ``` json
+    {
+      "source_omi_filter": {
+        "filters": {
+          "virtualization-type": "hvm",
+          "image-name": "image-name-in-account",
+          "root-device-type": "ebs"
+        },
+        "owners": ["099720109477"],
```

Comment:
This looks like it has been copy/pasted from the Amazon docs, since this is Cannonical's user number for Amazon. Is it possible to provide a valid filter so people can easily get an example running?